### DISCUSSION TITLE
min-width to use tailwind's spacing.

### DISCUSSION THREAD

**Discussion by 0xElectric** [tailwindlabs/tailwindcss#9538] [Ideas]
Why can't we use min-width with tailwind's spacing e.g. min-w-10?

Any thoughts about this?

**Comment by gungim** [comment]
default, min-width tailwindcss only support 
min-w-0 
min-w-full 
min-w-min 
min-w-max 
min-w-fit
if you should using other parameter, you can add them to tailwind.config file
